### ISSUE TITLE
Time displays should have aria-labels with written out times

### ISSUE THREAD

**Issue by claudiah12** [videojs/video.js#6336]
## Description
Times such as current time and duration are written in a timestamp format such as "0:20/14:05". Sighted users can parse this as time when they read it, but screen reader users will read this as simply numbers and punctuation rather than a human-understandable time with hours/minutes/seconds.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. Turn on screen reader software
2. Navigate to a timestamp such as the current time or duration
3. The screen reader will read the timestamp string as numbers and punctuation rather than an understandable time

### Expected
If seek bar is at 0:20/14:05, then screen reader announces 'twenty seconds of 14 minutes, 5 seconds

### Actual
If seek bar is at 0:20/14:05, then screen reader announces 'Zero colon zero colon twenty of fourteen colon one'.

## Additional Information
Related to: #6335

**Comment by OwenEdwards** [comment]
@claudiah12 see my comment about your other issue, #6335: https://github.com/videojs/video.js/issues/6335#issuecomment-615940053

**Comment by damanV5** [comment]
@gkatsev  Please assign this to me.

**Comment by mister-ben** [comment]
@damanV5 feel free to tackle this, such contributions are welcome! Please note the earlier linked comment about the localization challenges. It might be that `<time datetime="PT30M10">30:10</time>` could be helpful now, if there is sufficient support by screenreaders.

**Comment by damanV5** [comment]
@mister-ben  I have raised https://github.com/videojs/video.js/pull/8985 in an attempt to fix this issue.

Please let me know what do you think about this.
